website/content/v1.3/introduction/getting-started.md

```diff
@@ -251,28 +259,58 @@ install:
 
 to reflect `vda` instead of `sda`.
 
+### Customizing Machine Configuration
+
+The generated machine configuration provides sane defaults for most cases, but machine configuration
+can be modified to fit specific needs.
+
+Some machine configuration options are available as flags for the `talosctl gen config` command,
+for example setting a specific Kubernetes version:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --kubernetes-version 1.25.4 my-cluster https://192.168.64.15:6443
+```
+
+Other modifications are done with [machine configuration patches]({{< relref "../talos-guides/configuration/patching" >}}).
+Machine configuration patches can be applied with `talosctl gen config` command:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml --config-patch-control-plane @cni.patch my-cluster https://192.168.64.15:6443
```

Comment:
Not sure if we have it explicitly called out somewhere else, but might be worth adding a note about making sure to use the `@` to specify a local file.